.checkstyle/checkstyle.xml

```diff
@@ -109,7 +109,7 @@
         </module>
         <module name="JavaNCSS">
             <!-- default is 50 -->
-            <property name="methodMaximum" value="100"/>
+            <property name="methodMaximum" value="105"/>
```

Comment:
It's better to add the offending class to `suppressions.xml` than bump the limit for all classes.